LevelUp.Pos.ProposedOrders.Tests/UpdateSpendAmountTests.cs

```diff
@@ -32,8 +32,8 @@ public void UpdateSpend_WhenProposedOrderRequestIs_PayingTooMuch()
             int outstandingTotalOnCheck = 1000;
             int amountCustomerIsPaying = 1200;
 
-            ProposedOrderCalculator.CalculateAdjustedCustomerPaymentAmount(outstandingTotalOnCheck, amountCustomerIsPaying)
-                .Should()
+            ProposedOrderCalculator.SanitizeData(outstandingTotalOnCheck, 0, 0, amountCustomerIsPaying)
```

Comment:
Funny, I hadn't considered that all we were really doing on the `SpendAmount` in the past was sanitizing it, and distinguishing that from the actual business logic calculations for adjusting the tax and the exemption amounts.  👍